.circleci/scripts/release-create-gh-release

```diff
@@ -28,24 +28,24 @@ function install_github_cli ()
 
 current_commit_msg=$(git show -s --format='%s' HEAD)
 
-if grep --quiet '^Version v' <<< "$current_commit_msg"
+if [[ $current_commit_msg =~ Version[-[:space:]]v([[:digit:]]+.[[:digit:]]+.[[:digit:]]+) ]]
 then
+    tag="${BASH_REMATCH[1]}"
+
     install_github_cli
 
     printf '%s\n' 'Creating GitHub Release'
-    read -ra commit_words <<< "$current_commit_msg"
-    tag="${commit_words[1]}"
     release_body="$(awk -v version="${tag##v}" -f .circleci/scripts/show-changelog.awk CHANGELOG.md)"
     pushd builds
         hub release create \
             --attach metamask-chrome-*.zip \
             --attach metamask-firefox-*.zip \
-            --message "${commit_words[0]} ${commit_words[1]#v}" \
+            --message "Version ${tag##v}" \
```

Comment:
Small nitpick but we don't need to strip the `v` prefix here:

```suggestion
            --message "Version ${tag}" \
```

The 1st (0-indexed) match will be without the `v` as per the regex above.

Reply:
Oops, I had intended to capture the `v` as well. I'll fix the regex instead, otherwise line 46 is incorrect.